index.html

```diff
@@ -49,5 +52,144 @@ <h3>I am a Software Developer</h3>
 		<img src="images/Illustration.png" alt="shapes background image" class="png">
 	</section>
 
+	<section class="works">
+		<div class="container">
+			<h1>My Recent Works</h1>
```

Comment:
- [ ] For best practice and semantics, kindly have only one `h1` tag in your whole project. For other headings, use the other `hx` tags.  This is o for the following reasons :

- Accessibility: Screen readers use the heading structure of a webpage to provide an overview of the content and navigate through it. Multiple h1 tags on a page can create confusion and make it difficult for screen readers to provide a clear and logical hierarchy of the content.

- SEO: Search engines also use the heading structure of a webpage to understand its content and rank it in search results. Having multiple `h1` tags can dilute the importance and relevance of each heading, making it more difficult for search engines to determine the main topic or focus of the page

- Visual hierarchy: Headings are also used to create a visual hierarchy on a page, with larger headings indicating more critical content. Using a logical hierarchy of headings, you can help users quickly scan and understand the content on your page.